package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "galio-framework",
   "main": "src/index.js",
-  "version": "1.0.3",
+  "version": "1.1.0",
```

Comment:
We were thinking about this and we're actually going to change the version to 0.5 until we create more components, screens and just a better UX for our users.

Reply:
This was the ‘above comment’ I was mentioning @hetmann